includes/class-avatar.php

```diff
@@ -170,7 +63,7 @@ public static function avatar_stored_in_comment( $args, $comment ) {
 		}
 
 		// Simple method to prevent broken images
-		$args['extra_attr'] .= sprintf( ' onerror="this.onerror=null;this.src=\'%1$s\';this.srcset=\'%1$s\';"', self::get_default_avatar() );
+		$args['extra_attr'] .= sprintf( ' onerror="this.onerror=null;this.src=\'%1$s\';this.srcset=\'%1$s\';"', plugin_dir_url( dirname( __FILE__ ) ) . 'assets/img/mm.jpg' );
```

Comment:
I would recommend to use one of these https://github.com/pfefferle/wordpress-webmention/blob/main/webmention.php#L34 instead of `plugin_dir_url( dirname( __FILE__ ) )`.

Reply:
hmm, I think there is no one for URL. Maybe adding the URL one and use it here.